tests/periph_qdec/main.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2014-2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test for low-level QDEC drivers
+ *
+ * This test initializes all declared QDEC devices.
+ * It displays QDEC counters value each second.
+ *
+ * @author      Gilles DOFFE <gdoffe@gmail.com>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "periph/qdec.h"
+#include "xtimer.h"
+
+void handler(void *arg)
+{
+    (void)arg;
+    printf("INTERRUPT\n");
+}
+
+int main(void)
+{
+    uint8_t i = 0;
+    int32_t error = 0;
+    puts("Welcome into QDEC test program !");
```

Comment:
Maybe add some details about what this test application will do. This will be displayed in the terminal and can guide the user.